common/shlibs

```diff
@@ -16,6 +16,8 @@
 # PLEASE NOTE: when multiple packages provide the same SONAME, the first
 # one (order top->bottom) is preferred over the next ones.
 #
+libKF5KDEGamesPrivate.so.1 libkdegames-18.08.0_1
```

Comment:
are those used immediately by another package ?

Reply:
they are not required immediately